project/Dependencies.scala

```diff
@@ -16,11 +16,12 @@ object Dependencies {
   val specs2 = PlayImport.specs2 % "test"
   val awsWrap = "com.github.dwhjames" %% "aws-wrap" % "0.7.2"
   val awsDynamo = "com.amazonaws" % "aws-java-sdk-dynamodb" % "1.9.31"
+  val scalaz = "org.scalaz" %% "scalaz-core" % "7.1.1"
 
   //projects
 
   val apiDependencies = Seq(identityCookie, identityPlayAuth, scalaUri,
-    playWS, playCache, playFilters, awsWrap, awsDynamo,
+    playWS, playCache, playFilters, awsWrap, awsDynamo, scalaz,
```

Comment:
It's alright so long as you don't do huge wildcard imports!

```scala
import scalaz._
import Scalaz._
```

is a no-no if you want to keep a sane compile time.
Having said that, I'm a bit disappointed by Scalaz so far, so if at the end of it we don't get any benefits we can just remove it. For now it's a transitive dependency anyway so it doesn't have much impact.